vignettes/getting-started.Rmd

```diff
@@ -0,0 +1,156 @@
+
+This "Getting Started with rbabylon" vignette takes the user through some basic scenarios for modeling with NONMEM using `rbabylon`, introducing you to its basic workflow and functionality. 
+
+# Setup
+
+There is some initial setup to get `rbabylon` configured. First, load the library and make sure you have installed `babylon` (often aliased as `bbi`) and told `rbabylon` where to find it. **POINT TO BBI INSTALLATION INSTRUCTIONS, do these exist?**
+
+```{r bbi_exe_path}
+library(rbabylon)
+options('rbabylon.bbi_exe_path' = '/data/apps/bbi')
+```
+
+Next, initialize `babylon` by pointing it to a working installation of NONMEM, and default version to use.
+
+```{r setup, eval=FALSE}
+bbi_init(.dir = "../inst/nonmem",     # the modeling directory, where your control streams will be
+         .nonmem_dir = "/opt/NONMEM", # location of NONMEM installation
+         .nonmem_version = "nm74gf")  # default NONMEM version to use
+```
+
+Finally, it is best to set the modeling directory. By doing this, you do not need to pass the full path to model files, but only the path relative to the specified modeling directory. Pass a path relative to your current working directory to `set_model_directory()`
+
+```{r set_model_directory, echo = FALSE}
+set_model_directory("../inst/nonmem")
+```
+
+# Initial Modeling Run
+
+## Create Model Object
+To begin modeling, first create a model object. This is an S3 object with an accompanying YAML file, kept in your modeling directory. The YAML serves as the "record of truth" for the model, and will persist changes you make to the model as you go.
+
+The `.yaml_path` argument should have the same name (but with a `.yaml` extension) as your base model control stream. For instance, the command below assumes you have a control stream name `1.ctl` or `1.mod` in the model directory you just defined. The `.description` argument is required and corresponds to what would be in the `$PROB` definition in a control stream.
+
+```{r first model}
+mod1 <- new_model(.yaml_path = "1.yaml", .description = "our first model")
+```
+
+The model object you have just created can be passed to various functions which you will see in a minute. If you ever need to recreate it, just run `mod1 <- read_model("1.yaml")` to rebuild it from the YAML. The first thing we will do is submit the model to be run.
+
+## Submit Model
+
+```{r submit_model_real, include = FALSE}
+proc1 <- mod1 %>% submit_model(.dry_run = TRUE)
+```
+
+```{r submit_model_fake, eval = FALSE}
+proc1 <- mod1 %>% submit_model()
+```
+
+This will return a process object. We won't discuss this object in this vignette, but other documentation will show how it can be used to check on the model run in progress. Please note that, for this first release, **checking on a model run in progress is not fully implemented.** For now, users should check on their runs manually and only proceed to the next steps once it has successfully completed. 
```

Comment:
this might be too much detail for the intro vignette, can probably just show `mod101 %>% submit_model()` and leave it at that

Reply:
I like that